index.html

```diff
@@ -6,7 +6,411 @@
   </head>
   <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+    <header class="page-header">
 
+      <p class="page-header-paragraph">
+      <h1 class="visually-hidden">Интернет-магазин гаджетов device</h1>
+      <img class="logo-device" src="" alt="Логотип интернет-магазина гаджетов device">
+      </p>
+
+      <nav class="navigation">
+        <form class="navigation-form" action="https://echo.htmlacademy.ru/" method="post">
+          <label for="navigation-search">
+            <span class="visually-hidden">
+              Поиск
+            </span>
+          </label>
+          <input type="text" name="navigation-search" id="navigation-search" placeholder="Поиск по сайту">
+        </form>
+        <ul class="navigation-list navigation-user">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Войти</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Сравнить</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Корзина</a>
+          </li>
+        </ul>
+        <a class="navigation-link" href="catalog.html">Каталог товаров</a>
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Доставка</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Гарантия</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Контакты</a>
+          </li>
+      </nav>
+    </header>
+
+    <main class="main-index">
+      <section class="gallery">
+        <h2 class="gallery-title">Галлерея товаров</h2>
+        <img class="slider-image" src="" alt="Примеры товаров">
+        <button class="slider-button slider-prev" type="button">
+          <span class="visually-hidden">Предыдущий товар</span>
+        </button>
+        <button class="slider-button slider-next" type="button">
+          <span class="visually-hidden">Следующий товар</span>
+        </button>
+        <ol class="slider-pagination">
+          <li class="slider-pagination-item">
+            <button class="slider-pagination-button" type="button">
+              <span class="visually-hidden">1 фото</span>
+            </button>
+            <p class="slider-description">
+              <b class="slider-description-title">
+                Порхает как бабочка,<br>жалит как пчела!
+              </b>
```

Comment:
```suggestion
              <strong class="slider-description-title">
                Порхает как бабочка, жалит как пчела!
              </strong >
```

На этих "подзаголовков" делается прям смысловой акцент, поэтому `strong` больше подходит. `br` пока лучше не растравлять